src/organizer.js

```diff
@@ -63,7 +63,13 @@ class Organizer {
           console.log(`send successful for ${JSON.stringify(userA)} and ${JSON.stringify(userB)}`);
           return this.recordMeeting(userA, userB);
         })
-        .catch(() => {
+        .catch((error) => {
+          if (error && error.response && error.response.data) {
```

Comment:
Forgot to suggest [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) 😅

```suggestion
          if (error?.response?.data) {
```